AirSkyBoat/AirSkyBoat
# Fix usable equipment that gives enchantment status effect

**_I affirm:_**


## Please enter a player-facing description
Fix useable equipment that gives the enchantment status effect (for example High Breath Mantle). Also removing powder boots no longer removes the flee status effect.

## What does this pull request do? (Please be technical)
This PR fixes useable equipment that gives the enchantment status effect (for example High Breath Mantle). This is done by adding a check to the status effect container to use the itemsource field. Also removing powder boots no longer removes the flee status effect.

## Steps to test these changes
!additem 15320 and use powder boots then remove to see flee still active
!additem 15487 and use high breath mantle and see the change in HP

Closes https://github.com/HorizonFFXI/HorizonXI-Issues/issues/1264
Closes https://github.com/HorizonFFXI/HorizonXI-Issues/issues/1201
Closes https://github.com/HorizonFFXI/HorizonXI-Issues/issues/1260
Closes https://github.com/HorizonFFXI/HorizonXI-Issues/issues/1235
## Special Deployment Considerations
